packages/graphql_client/lib/models/public_ips.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:graphql_client/models/query_return_functions.dart';
+
+class PublicIpsInfo {
+  String id;
+  String? contractId;
+  String? ip;
+  String? gateway;
+
+  PublicIpsInfo({
+    required this.id,
+    this.contractId,
+    this.ip,
+    this.gateway,
+  });
+
+  factory PublicIpsInfo.fromJson(Map<String, dynamic> json) {
```

Comment:
Also here u should validate the fields u wanna access exists first, otherwise the code will crash